frontend/components/dynamic-badge-maker.js

```diff
@@ -61,6 +61,7 @@ export default function DynamicBadgeMaker({
         <option value="json">json</option>
         <option value="xml">xml</option>
         <option value="yaml">yaml</option>
+        <option value="html">html</option>
```

Comment:
Sure, but while they're currently sorted alphabetically, they're also sorted by general applicability in the sense that JSON and XML are ubiquitous and stuff used in APIs, YAML is a robust standard for interchange but less common, and grabbing stuff from HTML pages to badges is inherently possibly doing something that the document author did not intend it to be used like. So in that sense I think HTML deserves a place at the bottom.

If you still thing alphabetical sorting would be better, let me know and I'll change.

Reply:
Uh, well I see the order is different in the drop-down and the example URLs. The URLs order kind of makes sense so that XML and HTML are next to each other there. But in any case, the order should be the same in those two places, no strong opinions here so I'll just change to alphabetical in both.